.pre-commit-config.yaml

```diff
@@ -30,3 +30,23 @@ repos:
     rev: 23.7.0
     hooks:
       - id: black
+  - repo: https://github.com/gitleaks/gitleaks
```

Comment:
what are these?

Reply:
gitleaks repo is used to detect and prevent hardcoded secrets like passwords, API keys and tokens in git repositries.